swarm/storage/ldbstore_test.go

```diff
@@ -34,21 +34,26 @@ type testDbStore struct {
 	dir string
 }
 
-func newTestDbStore(mock bool) (*testDbStore, error) {
+func newTestDbStore(mock bool, trusted bool) (*testDbStore, error) {
 	dir, err := ioutil.TempDir("", "bzz-storage-test")
 	if err != nil {
 		return nil, err
 	}
 
 	var db *LDBStore
+	params := NewLDBStoreParams(dir, defaultDbCapacity, nil, nil)
```

Comment:
This would read so much better if it is just `NewLDBStoreParams(dir)`. If we must have a way to change some of the default params, we should do it in a separate constructor, or with chaining for example - I think both are better than passing `nil`s.